avm/src/vm/memory.rs

```diff
@@ -5,7 +5,7 @@ use crate::vm::program::Program;
 
 // -2^63
 pub const CLOSURE_ARG_MEM_START: i64 = -9223372036854775808;
-pub const CLOSURE_ARG_MEM_END: i64 = CLOSURE_ARG_MEM_START + 3;
+pub const CLOSURE_ARG_MEM_END: i64 = CLOSURE_ARG_MEM_START + 9001; // TODO: IT'S OVER 9000!
```

Comment:
can we add some more context about why this is 9001 right now?

Reply:
Because it's over 9000!?

What!? 9000!?

Reply:
I can add more, I just couldn't fit it after the constant so it'll have to be a new line.